### PR TITLE
feat: add `VoiceChannelEffectSend` event

### PR DESCRIPTION
**Pull request by advaith1**
**Please describe the changes this PR makes and why it should be merged:**
Adds the VoiceChannelEffectSend gateway event

**If applicable, please reference Discord API Docs PRs or commits that influenced this PR:**
* https://github.com/discord/discord-api-docs/pull/6025

### CODE REVIEW

File: deno/gateway/v10.ts [hunk 1]
```diff
@@ -1633,6 +1635,56 @@ export type GatewayUserUpdateDispatch = DataPayload<GatewayDispatchEvents.UserUp
...
+	 * The ID of the emoji animation, for emoji reaction effects
+	 */
+	animation_id?: number;
+}
+
+enum VoiceChannelEffectSendAnimationType {
+	/**
+	 * A fun animation, sent by a Nitro subscriber
+	 */
+	PREMIUM,
```
Comment (vladfrangu) [inline deno/gateway/v10.ts:1681]: We don't do screaming enums here

**Review by vladfrangu — DISMISSED** [review]

File: gateway/v10.ts [hunk 2]
```diff
@@ -1633,6 +1635,68 @@ export type GatewayUserUpdateDispatch = DataPayload<GatewayDispatchEvents.UserUp
...
+	 */
+	animation_id?: number;
+	/**
+	 * The ID of the soundboard sound, for soundboard effects
+	 */
+	sound_id?: Snowflake;
+	/**
+	 * The filename of the soundboard default sound, for soundboard effects
+	 */
+	sound_override_path?: string | null;
```
Comment (almeidx) [inline gateway/v10.ts:1682]: Not documented. Either mark as `@unstable` (if this is still included in the payload) or just get rid of it

File: gateway/v10.ts [hunk 3]
```diff
@@ -1633,6 +1635,68 @@ export type GatewayUserUpdateDispatch = DataPayload<GatewayDispatchEvents.UserUp
...
+	 * ID of the user who sent the effect
+	 */
+	user_id: Snowflake;
+	/**
+	 * The emoji sent, for emoji reaction and soundboard effects
+	 */
+	emoji?: APIEmoji | null;
+	/**
+	 * The type of emoji animation, for emoji reaction and soundboard effects
+	 * See https://discord.com/developers/docs/topics/gateway-events#voice-channel-effect-send-animation-types
```
Comment (almeidx) [inline gateway/v10.ts:1668]:
```suggestion
```

File: gateway/v10.ts [hunk 4]
```diff
@@ -1633,6 +1635,68 @@ export type GatewayUserUpdateDispatch = DataPayload<GatewayDispatchEvents.UserUp
...
+	 * The filename of the soundboard default sound, for soundboard effects
+	 */
+	sound_override_path?: string | null;
+	/**
+	 * The volume of the soundboard sound, from 0 to 1, for soundboard effects
+	 */
+	sound_volume?: number;
+}
+
+export enum VoiceChannelEffectSendAnimationType {
```
Comment (almeidx) [inline gateway/v10.ts:1689]:
```suggestion
/**
  * https://discord.com/developers/docs/topics/gateway-events#voice-channel-effect-send-animation-types
  */
export enum VoiceChannelEffectSendAnimationType {
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — APPROVED** [review]

**Pull request merged**